py/kubeflow/testing/cleanup_ci.py

```diff
@@ -117,6 +127,47 @@ def cleanup_endpoints(args):
   logging.info("Unexpired services:\n%s", "\n".join(unexpired))
   logging.info("expired services:\n%s", "\n".join(expired))
 
+
+def cleanup_disks(args):
+  credentials = GoogleCredentials.get_application_default()
+
+  compute = discovery.build('compute', 'v1', credentials=credentials)
+  disks = compute.disks()
+  next_page_token = None
+
+  expired = []
+  unexpired = []
+  unmatched = []
+
+  for zone in args.zones.split(","):
+    while True:
+      results = disks.list(project=args.project,
+                           zone=zone,
+                           pageToken=next_page_token).execute()
+      if not "items" in results:
+        break
+      for d in results["items"]:
+        name = d["name"]
+        if not is_match_disk(name):
+          unmatched.append(name)
+          continue
+
+        age = getAge(d["creationTimestamp"])
+        if age > datetime.timedelta(hours=args.max_age_hours):
+          logging.info("Deleting disk: %s, age = %r", name, age)
+          disks.delete(project=args.project, zone=zone, disk=name)
```

Comment:
Will disks.delete throw exception when disk can not be deleted? 
Like still used by a pod?
We might want to catch and continue here.

Reply:
there won't be exception thrown. but I updated the code to print out the response.

https://developers.google.com/resources/api-libraries/documentation/compute/v1/python/latest/compute_v1.disks.html#delete